.travis.yml

```diff
@@ -25,11 +25,8 @@ env:
   matrix:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
-    - EMBER_TRY_SCENARIO=ember-lts-2.4
-    - EMBER_TRY_SCENARIO=ember-lts-2.8
```

Comment:
For the time being, I'd like to leave 2.4 and 2.8 in the ember-try config so we remain aware of freestyle's compatibility in CI. We're only officially supporting the trailing 2 LTS releases (2.16 and 2.18 technically), so we'll include 2.4 and 2.8 in the `allow_failures` section below. Perhaps we should also add 2.12 as an allowed failure while we're at it.